internal/static/i18n/zh.yaml

```diff
@@ -0,0 +1,900 @@
+Errors:
+  Internal: 发生了内部错误
+  NoChangesFound: 没有变化
+  OriginNotAllowed: 这个"来源"是不被允许的
+  IDMissing: ID 丢失
+  ResourceOwnerMissing: 组织没有资源所有者
+  RemoveFailed: 无法移除
+  ProjectionName:
+    Invalid: 错误的映射名称
+  Assets:
+    EmptyKey: 资产的 Key 为空
+    Store:
+      NotInitialized: 资产存储未初始化
+      NotConfigured: 未配置资产存储
+    Bucket:
+      Internal: 创建桶时出现内部错误
+      AlreadyExists: 存储桶已存在
+      CreateFailed: 未创建存储桶
+      ListFailed: 无法读取存储桶
+      RemoveFailed: 未删除存储桶
+      SetPublicFailed: 无法将存储桶设置为公开
+    Object:
+      PutFailed: 未创建对象
+      GetFailed: 无法读取对象
+      NotFound: 找不到对象
+      PresignedTokenFailed: 无法创建签名令牌
+      ListFailed: 无法读取对象列表
+      RemoveFailed: 无法移除对象
+  Limit:
+    ExceedsDefault: 超出默认限制
+  Language:
+    NotParsed: 无法解析语言
+  OIDCSettings:
+    NotFound: OIDC 配置未找到
+    AlreadyExists: OIDC 配置已存在
+  SecretGenerator:
+    AlreadyExists: 秘密生成器已经存在
+    TypeMissing: 缺少秘钥生成器类型
+    NotFound: 未找到秘钥生成器
+  SMSConfig:
+    NotFound: 未找到 SMS 配置
```

Comment:
In some local software, similar computer proper nouns are not translated, because users do not know what they are after translation, and it is difficult for maintenance personnel to accurately locate them.

Reply:
thanks for clarification :)